### ISSUE TITLE
BUG: Enabling Darkmode makes slide presentations very laggy

### ISSUE THREAD

**Issue by Peanut1906** [zsviczian/obsidian-excalidraw-plugin#1807]
Hey i'd love to use the Excalidraw plugin but it is just way to laggy when darkmode is enabled. Especially when using the slideshow script. I have tested it in a completely new vault and i am experiencing the same issue there. Darkmode enabled => laggy, Darkmode not enabled => smooth. Is there something i could do about that?

Thanks in advance.
-Josh

**Comment by zsviczian** [comment]
This is interesting. Can you reproduce this with the default Obsidian theme and with other plugins disabled?
I tested this now on a large drawing / presentation, I did not notice any difference between light and dark mode.

**Comment by Peanut1906** [comment]
Yes i tried it with a completely new vault. no plugins except Excalidraw and the slideshow snippet. Heres a little video of the issue:

https://github.com/zsviczian/obsidian-excalidraw-plugin/assets/94468372/ca7d4e3d-15eb-440b-875a-bf771840a477

Btw i am using an Intel MacBook from 2020 with the latest version. To get around that problem i currently use the light mode but set the background to a dark color and the text to a light color. Not perfect but it absolutely works.
